src/sbt-test/sbt-1.5/cross-build-scala3/src/main/scala-3/Main.scala.expected

```diff
@@ -0,0 +1,5 @@
+import scala.Int
```

Comment:
this is not detected as a rename because I enabled removal of unused imports to make sure a rule was run (unrelated to the fix)